plugin/federation/federation.gotpl

```diff
@@ -6,6 +6,8 @@
 
 {{ reserveImport "github.com/99designs/gqlgen/plugin/federation/fedruntime" }}
 
+{{ .Imports }}
```

Comment:
This seems a little hacky. Trying to deal with imports in the context of code generation. Is there a better way?

Reply:
Yeah I think this has the potential to introduce tricky circular dependencies, which I see you worked around so that at least autgen models work.  That was actually one of the issues I saw before.  I think this complexity is from wrapping the resolver in the self executing function.